app/views/messages/homepage.erb

```diff
@@ -0,0 +1,26 @@
+<h1>Chitter</h1>
+<% if current_user %>
+	<form action='/message' method='post'>
+	  <label for='message'>Message:</label>
+	    <input type='text' name='peep'>
+	  <input type='submit' value='Post'>
+	</form>
+<% end %>
+<ul id='messages'>
+	<% @messages.each do |message| %>
+		<li>
+			<label id='username'>
+				<%= User.get(message.user_id).username %> 
```

Comment:
You should never refer to the model directly from the view, the controller should always act as an interface between your view and your model. In this case, you have passed @messages correctly. In order to get the username you could use `message.user.username`, assuming your relationships are correctly set up.